src/utils.js

```diff
@@ -607,3 +607,20 @@ export function useSessionStorage(key, initialValue) {
 
     return [storedValue, setValue];
 }
+
+/** 
+ * @description
```

Comment:
write documentation of when to use this function. what should be expected of "newData" and "cachedData" (i.e. what should the data types be for these params?)